src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,218 @@
+import { Component, Fragment } from 'react';
+import { Link } from 'react-router-dom';
```

Comment:
* The Github icon should be on the far right.
* The sandwich menu should be on the left since that's where the drawer will be coming from.